exporter/sumologicexporter/source_format.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sumologicexporter
+
+import "regexp"
+
+type sourceFormats struct {
+	name     sourceFormat
+	host     sourceFormat
+	category sourceFormat
+}
+
+type sourceFormat struct {
+	matches  [][]string
+	template string
+}
+
+func newSourceFormat(r *regexp.Regexp, text string) sourceFormat {
```

Comment:
Add comment what arguments does this `func` expect, i.e. what regex and what string and what are they used for.